src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,175 @@
+---
+linkTitle: Access control
+title: Access control for organizations in the web user interface
+description: Learn how to use our web UI for organization access control to quickly associate users, groups, and service accounts with organizations.
+weight: 20
+menu:
+  main:
+    identifier: orgs-accesscontrol
+    parent: web-orgs
+owner:
+  - https://github.com/orgs/giantswarm/teams/biscuit
+user_questions:
+  - How can I give someone access to an Organization in the web UI?
+  - How can I give someone access to the web UI?
+  - Where can I find the access control UI?
+last_review_date: 2021-04-20
+---
+
+# Access control for organizations in the web user interface
+
+Learn how to quickly grant and revoke access to an organization's resources via the web user interface, using Kubernetes role based access control (RBAC) mechanisms.
+
+{{% organization_mgmt_mapi_disclaimer %}}
+
+Granting access means that the users (when associating users or groups) or programs (when using service accounts) affected will be able to access resources via the Management API, including via the web user interface itself.
+
+The access control UI in the web interface provides quick access to a focussed set of RBAC resources, with the goal to make it easy for you to give someone – be it individuals or groups – or service accounts access to resources in an organization's namespace.
+
+## Where to find it {#where-to-find}
+
+1. Log in to the web user interface, using the URL matching your management cluster/installation.
+2. Click **Organizations** in the main menu in the top.
+3. Click the organization you want to manage access for.
+4. Select the **Access control** tab.
+
+## User interface overview {#ui-overview}
+
+![Access control user interface overview](access-control-ui.png)
+
+The access control user interface provides:
+
+- A list of roles, both from the organization's namespace and cluster roles, to select from. Each role defines a specific set of permissions for a number of resources. Cluster roles are marked using a 🌐 icon. In the list, it is also indicated how many users and groups are bound to each role in this organization's namespace.
```

Comment:
> A list of roles, both from the organization's namespace and cluster roles, to select from.

what do you think about

> A list of roles to select from, sourced from both the organization's namespace and cluster roles

? to avoid the embedded clause (which i always find challenging as a reader, especially for technical content)

Reply:
Good!